pay-api/src/pay_api/services/fas/routing_slip.py

```diff
@@ -167,6 +173,61 @@ def search(cls, search_filter: Dict, page: int, limit: int, return_all: bool = F
 
         return data
 
+    @classmethod
+    @user_context
+    def create_daily_reports(cls, day: str, **kwargs):
+        """Create and return daily report for the day provided."""
+        routing_slips: List[RoutingSlipModel] = RoutingSlipModel.search(
+            dict(
+                dateFilter=dict(
+                    endDate=day,
+                    startDate=day,
+                    target='created_on'
+                )
+            ),
+            page=1, limit=0, return_all=True
+        )[0]
+
+        total: float = 0
+        no_of_cash: int = 0
+        no_of_cheque: int = 0
+        total_cash_usd: float = 0
+        total_cheque_usd: float = 0
+        total_cash_cad: float = 0
+        total_cheque_cad: float = 0
+        # TODO Only CAD supported now, so just add up the total.
+        for routing_slip in routing_slips:
```

Comment:
should we consider parent - child here? i am not sure whats the ask here...

Reply:
If during the linking, total is not getting changed then all good. Report is for the routing slips created on a specific day.